.github/workflows/daily-image-tests.yml

```
@@ -0,0 +1,54 @@
+name: Daily broken images check
+
+on:
+  schedule:
+    # Every Weekday at 8 AM AEDT - https://cron.help/#0_21_*_*_SUN-THU
+    - cron: "0 21 * * SUN-THU"
+  workflow_dispatch:
+    inputs:
+      deploy_url:
+        description: "The URL of the site to test"
+        required: true
+      tests_to_run:
+        description: "The tests to run"
+        required: true
+      create_issue:
+        description: "Create an issue if there are broken images"
+        type: boolean
+        default: true
+        required: true
+
+defaults:
+  run:
+    shell: pwsh
+
+jobs:
+  check-broken-images:
+    name: Run Playwright Tests
+    uses: ./.github/workflows/template-ui-tests.yml
+    with:
+      deploy_url: ${{ inputs.deploy_url || 'https://ssw.com.au' }}
+      tests_to_run: ${{ inputs.deploy_url || 'images'}}
+
+  create-an-issue:
+    name: GitHub Issue
+    runs-on: ubuntu-latest
+    needs: check-broken-images
+    if: ${{ inputs.create_issue == true && needs.check-broken-images.outputs.testPassed == 'False' }}
```

```suggestion
    if: ${{ failed() && inputs.create_issue == true }}
```
I would've expected it to look like this. The previous job would fail which implies that the tests have failed

---

I think, it's failure() function, done

---

done